aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -576,6 +582,9 @@ Tensor ifft(const Tensor& self, const int64_t signal_ndim, const bool normalized
 
 Tensor rfft(const Tensor& self, const int64_t signal_ndim, const bool normalized,
             const bool onesided) {
+  TORCH_WARN_ONCE(
```

Comment:
Deprecating `torch.fft()` is the priority (since its name conflicts with the module), but deprecating these is OK if the same pattern is followed. I.e.:

""The function torch.rfft is deprecated and will be removed in a future PyTorch release. Use the new torch.fft module functions, instead, by importing torch.fft and calling torch.fft.rfft or torch.fft.rfftn."

And also updating the torch.rfft and torch.irfft docs.